src/particles/diagnostics/ReducedBeamCharacteristics.H

```diff
@@ -0,0 +1,61 @@
+/* Copyright 2022-2023 The Regents of the University of California, through Lawrence
+ *           Berkeley National Laboratory (subject to receipt of any required
+ *           approvals from the U.S. Dept. of Energy). All rights reserved.
+ *
+ * This file is part of ImpactX.
+ *
+ * Authors: Marco Garten, Chad Mitchell, Axel Huebl
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef IMPACTX_REDUCED_BEAM_CHARACTERISTICS_H
+#define IMPACTX_REDUCED_BEAM_CHARACTERISTICS_H
+
+#include "particles/ImpactXParticleContainer.H"
+#include "ImpactX.H"
+
+#include <AMReX_Extension.H>
+#include <AMReX_REAL.H>
+#include <AMReX_GpuComplex.H>
+
+#include <cmath>
+
+using namespace amrex;
```

Comment:
Do not use namespaces in `.H` files - this will "pollute" the include space of every downstream file that includes it.
```suggestion
```

Please only use this in `.cpp` files and even better, only in `{ ... }` sections of functions when needed at all.

Reply:
The only thing we really use in `{ ... }` is `using namespace amrex::literals; // for _rt and _prt`. The rest, please always write `amrex::` for clarity.